### PR TITLE
alpine: support edge vulnerabilities

### PR DESCRIPTION
**Pull request by RTann**
Followup to #1091 to support Alpine edge vulnerabilities

### CODE REVIEW

**Comment by RTann** [comment]
I updated my local Clair repo to use this version of ClairCore, and I ran the matcher. I found logs showing it was running the Alpine edge updater

File: alpine/distributionscanner.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ const (
 const (
 	issuePath = `etc/issue`
 
+	edgeVersion    = `edge`
```
Comment (crozzy) [inline alpine/distributionscanner.go:35]: can we consolidate both of these `const`s?

**Review by crozzy — COMMENTED** [review]
LGTM, just one nit

File: alpine/distributionscanner.go (hunk 1 above)
Comment (RTann) [reply alpine/distributionscanner.go:35]: not sure I follow

Comment (RTann) [reply alpine/distributionscanner.go:35]: @crozzy do you have any updates?

Comment (crozzy) [reply alpine/distributionscanner.go:35]: I meant have one `const` block instead of 2

Comment (RTann) [reply alpine/distributionscanner.go:35]: oh didn't see that, will do

Comment (RTann) [reply alpine/distributionscanner.go:32]: should be good now

**Review by hdonnay — APPROVED** [review]

**Pull request merged**